<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
          "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="description" content="Website of the Cantu Lab at UC Davis" />
  <meta name="keywords" content="UC Davis, Viticulture &amp; Enology, Grape, Genomics" />
  <meta name="author" content="Dario Cantu" />
  <link rel="stylesheet" type="text/css" href="styleCantupeople.css" title="Cantu Lab at UC Davis" media="screen,projection"/>
  <title>Cantu Lab at UC Davis</title>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-36299142-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body>
  <div id="wrap">
  <h1><a href="index.html">Cantu Lab at UC Davis</a></h1>
  <a class="slogan" href="http://wineserver.ucdavis.edu/"  target="_blank">Department of Viticulture &amp; Enology</a>
  
  <div id="topmenu">
    <p>
      <strong class="hide">Main menu:</strong>
      <a href="index.html">Home</a><span class="hide"> | </span>
      <a href="people.html">People</a><span class="hide"> | </span>
      <a href="projects.html">Projects</a><span class="hide"> | </span>
      <a class="active" href="data.html">Data</a><span class="hide"> | </span>
      <a href="http://scholar.google.com/citations?hl=en&user=p50Pg0gAAAAJ&sortby=pubdate&view_op=list_works&pagesize=100" target="_blank">Publications</a><span class="hide"> | </span>
    </p>
  </div>
	
  <div id="headerimage">
    <img src="header.jpg" width="980" height="200" alt="sample header image" />
  </div>

  <div id="sidebar">
    <img src="noble_rot.jpg" width="200" height="235" alt="noble rot"/>
    <img src="pm.jpg" width="200" height="231" alt="powdery mildew berries"/>
  </div>
  
  <div id="content">
    <div class="full">
      <br/>
      <h5>Data</h5>
    </div>
    <div class="all">
    <div class="boxed">
     <p><img src="cabernet.jpg" width="250" height="333" alt="cabernet"/>
     <h4><i>Vitis vinifera </br> cv. Cabernet Sauvignon</i></h4><strong> <br>Diploid contig assembly</strong> (<a href="http://www.nature.com/nmeth/journal/v13/n12/full/nmeth.4035.html" target="_blank">Chin et al., 2016 </a> & <a href="https://www.frontiersin.org/articles/10.3389/fpls.2017.00826/full" target="_blank"> Minio et al., 2017</a>)
     <br>NCBI Bioproject (SRA data): <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA316730"  target="_blank">PRJNA316730</a>   
        <p>
	<p><strong>Primary contigs:</strong> <br> 718 contigs; total size: 591,420,921 bp; N50: 4,372,253 bp
	<br>Download (Fasta is divided in 8 files, need to concatenated):
    <br>CS8_74SMRTcells_FALCON-unzip_primary.fasta - <a href="CS8_74SMRTcells_FALCON-unzip_primary_1.fasta.gz">1 </a>
    <a href="CS8_74SMRTcells_FALCON-unzip_primary_2.fasta.gz">2 </a>
    <a href="CS8_74SMRTcells_FALCON-unzip_primary_3.fasta.gz">3 </a> 
    <a href="CS8_74SMRTcells_FALCON-unzip_primary_4.fasta.gz">4 </a>
    <a href="CS8_74SMRTcells_FALCON-unzip_primary_5.fasta.gz">5 </a>
    <a href="CS8_74SMRTcells_FALCON-unzip_primary_6.fasta.gz">6 </a>
    <a href="CS8_74SMRTcells_FALCON-unzip_primary_7.fasta.gz">7 </a>
    <a href="CS8_74SMRTcells_FALCON-unzip_primary_8.fasta.gz">8 </a>    
	<br><br><strong>Haplotigs:</strong> <br> 2,037 contigs; total size: 367,781,855 bp; N50: 779,223 bp 
	<br>Download (Fasta is divided in 2 files, need to be concatenated):
    <br>CS8_74SMRTcells_FALCON-unzip_haplotigs.fasta - <a href="CS8_74SMRTcells_FALCON-unzip_haplotigs_1.fasta.gz">1 </a>
    <a href="CS8_74SMRTcells_FALCON-unzip_haplotigs_2.fasta.gz">2 </a>	
    <br><br><strong>Annotation</strong> (<a href="" target="_blank">Minio et al., 2018 BIORXIV/2018/269530</a>)
        <br> Protein coding genes coordinates (<a href="CS_clean.primary.corrected_loci.gff3.gz">gff3</a>); 
        Repeats (<a href="CS.repeats.primary.gff.gz">gff</a>); 
        CDS (<a href="CS_clean.primary.corrected_loci.CDS.fasta.gz">fasta</a>);
        Proteins (<a href="CS_clean.primary.corrected_loci.prot.faa.gz">fasta</a>); 
        Iso-Seq data (<a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRP132320" target="_blank">SRP132320</a>)
       <br style="clear:both;" />
     <br>
	</p>          
     </div>
     </br>
    <div class="boxed">
     <p><img src="phomopsis.jpg" width="250" height="320" alt="phomopsis"/>
     <h4><i>Diaporthe ampelina</i></h4><strong>Draft Genome</strong> (isolate DA912)
     <br>collaboration with <a href="http://ucanr.edu/sites/Rolshausen/"  target="_blank">Philippe Rolshausen</a> and <a href="http://ucanr.edu/?facultyid=2860"  target="_blank">Kendra Baumgartner</a></p>
	<p>Assembly size: 47.3 Mb (gaps: 72 kb; n.scaffolds: 2,383)  
	<br>N50:  132 Kb  
	<br>L50:  103  
	<br>Predicted protein coding genes:  10,704
	</p>
     <br>Scaffolds: <a href="DA912_scaffolds.v0.1.fasta.gz">DA912_scaffolds.v0.1.fasta.gz</a>
     <br>Peptides (<i>ab initio</i>):  <a href="DA912_peptides.v0.2.fasta.gz">DA912_peptides.v0.2.fasta.gz</a>
     <br>Transcripts (<i>ab initio</i>):   <a href="DA912_transcripts.v0.2.fasta.gz">DA912_transcripts.v0.2.fasta.gz</a>
     <br>WGS Accession: <a href="http://www.ncbi.nlm.nih.gov/nuccore/LCUC00000000"  target="_blank">LCUC00000000</a>	
     <br>Citation: Morales-Cruz A. et al., 2015. <a href="http://www.biomedcentral.com/1471-2164/16/469" target="_blank">BMC Genomics 16: 469 </a>
       <br style="clear:both;" />
     <br>
	</p>          
     </div>
     <br>
     <div class="boxed">
     <p><img src="diplodia_seriata.jpg" width="250" height="311" alt="diplodia_seriata"/>
     <h4><i>Diplodia seriata</i></h4><strong>Draft Genome</strong> (isolate DS831)
     <br>collaboration with <a href="http://ucanr.edu/sites/Rolshausen/"  target="_blank">Philippe Rolshausen</a> and <a href="http://ucanr.edu/?facultyid=2860"  target="_blank">Kendra Baumgartner</a></p>
	<p>Assembly size: 37 Mb (gaps: 72 kb; n.scaffolds: 694)  
	<br>N50:  304 Kb  
	<br>L50:  39  
	<br>Predicted protein coding genes:  9,343
	</p>
     <br>Scaffolds: <a href="DS831_scaffolds-v1.0.fasta.gz">DS831_scaffolds-v1.0.fasta.gz</a>
     <br>Peptides (<i>ab initio</i>):  <a href="DS831_peptides-v2.0.fasta.gz">DS831_peptides-v2.0.fasta.gz</a>
     <br>Transcripts (<i>ab initio</i>):   <a href="DS831_transcripts-v2.0.fasta.gz">DS831_transcripts-v2.0.fasta.gz</a>
     <br>WGS Accession: <a href="http://www.ncbi.nlm.nih.gov/nuccore/LAQI00000000"  target="_blank">LAQI00000000</a>	
     <br>Citation: Morales-Cruz A. et al., 2015. <a href="http://www.biomedcentral.com/1471-2164/16/469" target="_blank">BMC Genomics 16: 469 </a>
       <br style="clear:both;" />
     <br>
	</p>          
     </div>
     <br>
    <div class="boxed">
     <p><img src="pm.jpg" width="285" height="330" alt="Erysiphe necator"/>
     <h4><i>Erysiphe necator</i></h4>
     <br>Reference: <a href="http://www.biomedcentral.com/1471-2164/15/1081">Jones et al., 2014  BMC Genomics 15:1081 </a>
	<p>Mean assembly size: 50.5 ± 1.2 Mb (Est. genome size: 126.5 ± 18.4 Mb)  
	<br>Mean N50:  17.1 ± 3.1 Kb  
	<br>Mean L50:  897.2 ± 73.7 scaffolds  
	<br>Predicted protein coding genes:  6,533
	</p>
	 <br>Gene models (C-strain): <a href="e-necator.c-strain.gff.gz">e-necator.c-strain.gff.gz</a>
	 <br>Transcripts (C-strain): <a href="E.necator.transcripts.NCBI.6533.fasta.gz">e.necator.transcripts.NCBI.6533.fasta.gz</a>
	 <br>Peptides (C-strain):  <a href="E.necator.proteins.NCBI.6533.fasta.gz">e.necator.proteins.NCBI.6533.fasta.gz</a>
	 <br></br>	
     <br>Scaffolds (C-strain): <a href="e-necator.scaffolds.c.NCBI.fasta.gz">e-necator.scaffolds.c.NCBI.fasta.gz</a>
     <br>Scaffolds (Lodi): <a href="e-necator.scaffolds.lodi.NCBI.fasta.gz">e-necator.scaffolds.lodi.NCBI.fasta.gz</a>
     <br>Scaffolds (branching): <a href="e-necator.scaffolds.branching.NCBI.fasta.gz">e-necator.scaffolds.branching.NCBI.fasta.gz</a>
     <br>Scaffolds (e1-101): <a href="e-necator.scaffolds.e1-101.NCBI.fasta.gz">e-necator.scaffolds.e1-101.NCBI.fasta.gz</a>
     <br>Scaffolds (Ranch9): <a href="e-necator.scaffolds.ranch-9.NCBI.fasta.gz">e-necator.scaffolds.ranch-9.NCBI.fasta.gz</a>
       <br style="clear:both;" />
     <br> 
     <br>
	</p>          
     </div>
     <br>
     <div class="boxed">
     <p><img src="esca.jpg" width="250" height="330" alt="esca"/>
     <h4><i>Togninia minima</i></h4><strong>Draft Genome</strong> (isolate UCR-PA7) - collaboration with <a href="http://ucanr.edu/sites/Rolshausen/"  target="_blank">Philippe Rolshausen</a></p>
	<p>Assembly size: 47.5 Mb (gaps: 83 kb; n.scaffolds: 624) 
	<br>Median coverage: 103X  
	<br>N50:  336 Kb  
	<br>L50:  45  
	<br>G+C:  49.66%
	<br>Predicted protein coding genes:  8,926
	</p>
     <br>Scaffolds: <a href="PA7_scaffolds.v0.4.fasta.gz">PA7_scaffolds.v0.4.fasta.gz</a>
     <br>Contigs: <a href="PA7_contigs.v0.4.fasta.gz">PA7_contigs.v0.4.fasta.gz</a>
     <br>Peptides (<i>ab initio</i>):  <a href="PA7_proteins.v0.4.fasta.gz">PA7_proteins.v0.4.fasta.gz</a>
     <br>Transcripts (<i>ab initio</i>):   <a href="PA7_transcripts.v0.4.fasta.gz">PA7_transcripts.v0.4.fasta.gz</a>
     <br>WGS Accession: <a href="http://www.ncbi.nlm.nih.gov/nuccore/AORD00000000"  target="_blank">AORD00000000</a>	
     <br>Raw reads:  <a href="http://www.ncbi.nlm.nih.gov/sra/?term=SRX220948"  target="_blank">Short Read Archive</a>
     <br>Citation: <a href="http://genomea.asm.org/content/1/3/e00390-13.long">Blanco-Ulate et al., 2013 Genome Announcements 1(3):e00390-13</a>
       <br style="clear:both;" />
     <br>
	</p>          
     </div>
     <br>
     <div class="boxed">
     <p><img src="phaeomoniella_chlamydospora.jpg" width="250" height="320" alt=phaeomoniella_chlamydospora"/>
     <h4><i>Phaeomoniella chlamydospora</i></h4><strong>Draft Genome</strong> (isolate UCR-PC4)
     <br>collaboration with <a href="http://ucanr.edu/sites/Rolshausen/"  target="_blank">Philippe Rolshausen</a> and <a href="http://ucanr.edu/?facultyid=2860"  target="_blank">Kendra Baumgartner</a></p>
	<p>Assembly size: 27.5 Mb (gaps: 18 kb; n.scaffolds: 702)  
	<br>N50:  179 Kb  
	<br>L50:  50  
	<br>Predicted protein coding genes:  10,704
	</p>
     <br>Scaffolds: <a href="UCRPC4_scaffolds.v0.1.fasta.gz">UCRPC4_scaffolds.v0.1.fasta.gz</a>
     <br>Peptides (<i>ab initio</i>):  <a href="UCRPC4_peptides.v0.2.fasta.gz">UCRPC4_peptides.v0.2.fasta.gz</a>
     <br>Transcripts (<i>ab initio</i>):   <a href="UCRPC4_transcripts.v0.2.fasta.gz">UCRPC4_transcripts.v0.2.fasta.gz</a>
     <br>WGS Accession: <a href="http://www.ncbi.nlm.nih.gov/nuccore/LCWF00000000"  target="_blank">LCWF00000000</a>	
     <br>Citation: Morales-Cruz A. et al., 2015. <a href="http://www.biomedcentral.com/1471-2164/16/469" target="_blank">BMC Genomics 16: 469 </a>
       <br style="clear:both;" />
     <br>
	</p>          
     </div>
     <br>
     <div class="boxed">
     <p><img src="np2.jpg" width="250" height="330" alt="Neofusicoccum"/>
     <h4><i>Neofusicoccum parvum</i></h4><strong>Draft Genome</strong> (isolate UCR-NP2) - collaboration with <a href="http://ucanr.edu/sites/Rolshausen/"  target="_blank">Philippe Rolshausen</a></p>
       <p>Assembly size: 42.5  Mb (gaps: 69 kb; n.scaffolds: 1,287)
       <br>Median coverage: 113X
       <br>N50:  83 Kb
       <br>L50:  149
       <br>G+C:  56.7%
       <br>Predicted protein coding genes:  10,470
       </p>	
       <br>Scaffolds: <a href="NP2_scaffolds.v0.4.fasta.gz">NP2_scaffolds.v0.4.fasta.gz</a>
       <br>Contigs: <a href="NP2_contigs.v0.4.fasta.gz">NP2_contigs.v0.4.fasta.gz</a>
       <br>Peptides (<i>ab initio</i>):  <a href="NP2_proteins.v0.4.fasta.gz">NP2_proteins.v0.4.fasta.gz</a>
       <br>Transcripts (<i>ab initio</i>):   <a href="NP2_transcripts.v0.4.fasta.gz">NP2_transcripts.v0.4.fasta.gz</a>
       <br>WGS Accession: <a href="http://www.ncbi.nlm.nih.gov/nuccore/AORE00000000"  target="_blank">AORE00000000</a>
       <br>Raw reads:  <a href="http://www.ncbi.nlm.nih.gov/sra?term=SRX220832"  target="_blank">Short Read Archive</a>
       <br>Citation: <a href="http://genomea.asm.org/content/1/3/e00339-13.full?sid=5190b1fc-5e45-4b34-9e5f-e95e02c0f574">Blanco-Ulate et al., 2013 Genome Announcements 1(3):e00339-13</a>
       <br style="clear:both;" />
      <br>
        </p>
      </div> 
     <br>
	<div class="boxed">
     <p><img src="eutypa.jpg" width="250" height="320" alt="eutypa"/>
     <h4><i>Eutypa lata</i></h4><strong>Draft Genome</strong> (isolate UCR-EL1) - collaboration with <a href="http://ucanr.edu/sites/Rolshausen/"  target="_blank">Philippe Rolshausen</a></p>
	<p>Assembly size: 54.0 Mb (gaps: 103 kb; n.scaffolds: 2,334) 
	<br>Median coverage: 97X  
	<br>N50:  68.3 kb  
	<br>L50:  238  
	<br>G+C:  46.6%
	<br>Predicted protein coding genes:  11,818
	</p>
     <br>Scaffolds: <a href="EL1_scaffolds.v0.4.fasta.gz">EL1_scaffolds.v0.4.fasta.gz</a>
     <br>Contigs: <a href="EL1_contigs.v0.4.fasta.gz">EL1_contigs.v0.4.fasta.gz</a>
     <br>Peptides (<i>ab initio</i>):  <a href="EL1_proteins.v0.4.fasta.gz">EL1_proteins.v0.4.fasta.gz</a>
     <br>Transcripts (<i>ab initio</i>):   <a href="EL1_transcripts.v0.4.fasta.gz">EL1_transcripts.v0.4.fasta.gz</a>
     <br>WGS Accession: <a href="http://www.ncbi.nlm.nih.gov/nuccore/AORF00000000" target="_blank">AORF00000000</a>	
     <br>Raw reads:  <a href="http://www.ncbi.nlm.nih.gov/sra?term=SRX220831"  target="_blank">Short Read Archive</a>
     <br>Citation: <a href="http://genomea.asm.org/content/1/3/e00228-13.full"  target="_blank">Blanco-Ulate et al., 2013 Genome Announcements 1(3) e00228-13</a>			
     <br style="clear:both;" />
     <br>
	</p>          
     </div>
     <br>
     <div class="boxed">
	<p><img src="bcdw1.jpg" width="250" height="342" alt="bcdw1"/>
        <h4><i>Botrytis cinerea</i></h4><strong>Draft Genome</strong> (isolate BcDW1) </p>
	<p>Assembly size: 42.1 Mb (gaps: 58 kb; n.scaffolds: 453; median coverage: 125.3X)  
	<br>N50:  194 kb  
	<br>L50:  69  
	<br>G+C:  42%
	<br>Predicted protein coding genes:  11,073
	</p>
     <br>Scaffolds: <a href="BcDW1_scaffolds.v0.42.fasta.gz">BcDW1_scaffolds.v0.42.fasta.gz</a>
     <br>Contigs: <a href="BcDW1_contigs.v0.42.fasta.gz">BcDW1_contigs.v0.42.fasta.gz</a>
     <br>Peptides (<i>ab initio</i>):  <a href="BcDW1_proteins-v0.1.fasta.gz">BcDW1_proteins-v0.1.fasta.gz</a>
     <br>Transcripts (<i>ab initio</i>):   <a href="BcDW1_transcripts-v0.1.fasta.gz">BcDW1_transcripts-v0.1.fasta.gz</a>
     <br>WGS Accession: <a href="http://www.ncbi.nlm.nih.gov/nuccore/AORW00000000"  target="_blank">AORW00000000</a>	
     <br>Raw reads:  <a href=http://www.ncbi.nlm.nih.gov/sra/?term=SRX227202" target="_blank">Short Read Archive</a>
     <br>Citation: <a href="http://genomea.asm.org/content/1/3/e00252-13.full" target="_blank">Blanco-Ulate et al., 2013 Genome Announcements 1(3) e00252-13</a>			
     <br style="clear:both;" />
     <br>
	</p>          
     </div> 
     <br>
     	<div class="boxed">
     <p><img src="pst2.jpg" width="250" height="290" alt="pst"/>
     <h4><i>Puccinia striiformis</i> f.sp. <i>tritici</i></h4><strong>Draft genomes</strong> - collaboration with <a href="http://www.plantsciences.ucdavis.edu/dubcovsky/"  target="_blank">Jorge Dubcovsky</a> and <a href="http://www.jic.ac.uk/profile/cristobal-uauy.asp" target="_blank">Cristobal Uauy</a></p>
     <p>Pst-21: <a href="http://www.ncbi.nlm.nih.gov/nuccore/AORR00000000"  target="_blank">AORR00000000</a>
     <br>Pst-43: <a href="http://www.ncbi.nlm.nih.gov/nuccore/AORQ00000000"  target="_blank">AORQ00000000</a>
     <br>Pst-130: <a href="http://www.ncbi.nlm.nih.gov/nuccore/AEEW00000000"  target="_blank">AEEW00000000</a>	
     <br>Pst-87/7: <a href="http://www.ncbi.nlm.nih.gov/nuccore/AORT00000000"  target="_blank">AORT00000000</a>
     <br>Pst-08/21: <a href="http://www.ncbi.nlm.nih.gov/nuccore/AORS00000000"  target="_blank">AORS00000000</a>
     <br><i>Ab initio</i> gene models are available from  <a href="http://www.yellowrust.com"  target="_blank">yellowrust.com</a>
     </p>    	 
     <br>Citations: 
     <br><a href="http://www.plosone.org/article/info%3Adoi/10.1371/journal.pone.0024230"  target="_blank">Cantu et al., 2011 PLoS ONE</a>      
     <br><a href="http://www.biomedcentral.com/1471-2164/14/270/abstract"  target="_blank">Cantu et al., 2013 BMC Genomics</a>
     </div>
     <br>
     <br/>
     <br style="clear:both;" />	
     </div>
     <hr class="clear" />
     </div>
  <div class="span-24" id="footer">
    <p style="text-align: center;"><strong>Cantu Lab</strong> is proudly powered by <a href="https://github.com/"  target="_blank"><strong>Github</strong></a> - Template design by <a href="http://andreasviklund.com/">Andreas Viklund</a><br/> 
      <strong>Copyright (c) Dario Cantu, 2017</strong><br/>

      This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/">Creative
      Commons Attribution-ShareAlike 3.0 Unported License</a>.<br/>
      Please notify me if you use my content.
      <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/"><img alt="Creative Commons License" style="border-width:0" src="http://i.creativecommons.org/l/by-sa/3.0/80x15.png" /></a><br/>
    </p>
  </div>
  </div><!--id:wrap end-->
</body>
</html>
